.gitignore

```diff
@@ -1,2 +1,3 @@
 .resources
-*~
\ No newline at end of file
+*~
+.idea/
```

Comment:
What is `.idea/`?

Reply:
IntelliJ/Goland etc creates this when you open a project